README.md

```diff
@@ -5,7 +5,11 @@ Servable is an implementation of a "simple" observable
 ## To install run:
 
 ```
-npm i servable --save
+npm i servable
```

Comment:
Removed the `--save` did ja? ;-)

Reply:
I'm too young to know what `--save` was 😜